tests/test-sources/plugins/languages/markdown/markview.nix

```diff
@@ -0,0 +1,50 @@
+_: {
+  empty = {
+    plugins.markview.enable = true;
+  };
+
+  defaults = {
+    plugins.markview = {
+      enable = true;
+
+      settings = {
+        buf_ignore = [ "nofile" ];
+        mode = [
+          "n"
+          "no"
+        ];
+
+        hybrid_modes = [ ];
+
+        callback = {
+          on_enable = # Lua
+            ''
+              function(buf, win)
+                vim.wo[window].conceallevel = 2;
+                vim.wo[window].concealcursor = "nc";
+              end
+            '';
+
+          on_disable = # Lua
+            ''
+              function(buf, win)
+                  vim.wo[window].conceallevel = 0;
+                  vim.wo[window].concealcursor = "";
+              end
+            '';
+
+          on_mode_change = # Lua
+            ''
```

Comment:
I assume this was `on_mode_change = /* lua */ ''` but was affected by https://github.com/NixOS/nixfmt/issues/179?

Reply:
Yeah they change it to the single line like that… I don’t like it.